src/getBucketsForNode.ts

```diff
@@ -7,12 +9,18 @@
  *
  * @param node - The node which buckets should be retrieved
  */
-export default function getBucketsForNode(node: Node): string[] {
+export default function getBucketsForNode(
+	node:
```

Comment:
As discussed, we should leave a todo here noting we should replace this with our own node type when introduce it.